.travis.yml

```diff
@@ -13,5 +18,16 @@ before_install:
   - export PATH="$HOME/google-cloud-sdk/platform/cloud-datastore-emulator:$PATH"
   - cloud_datastore_emulator create tmp/local_datastore
```

Comment:
I don't think we need to start the dev local_datastore on 8180. Take a look within the test_helper.rb, the tests run against an in-memory instance of the emulator which is started on port 8181.